lib/vendor/js-cookies.js

```diff
@@ -26,6 +26,8 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 **/
 
+/* eslint-disable no-useless-escape */
```

Comment:
This is for the regular expressions?

Reply:
Yeah, I wanted to limit edits to the file